dashboard/app/views/teacher_dashboard/show.html.haml

```diff
@@ -11,6 +11,8 @@
   teacher_dashboard_data[:userName] = @current_user.short_name
   teacher_dashboard_data[:validGrades] = @valid_grades
   teacher_dashboard_data[:localeCode] = request.locale
+  teacher_dashboard_data[:textToSpeechScriptIds] = Script.text_to_speech_script_ids
```

Comment:
How are these used?  (I'm wondering if this should be two things or just one thing with the pre-reader scripts included in `:testToSpeechScriptIds`.)

Reply:
I think we will still want access to both `pre_reader_script_ids` and `text_to_speech_script_ids` because `text_to_speech_script_ids` are used to determine whether it's even possible to turn on tts autoplay (whether the toggle should show in the section editor) and `pre_reader_script_ids` are used to determine whether it should be on by default